extensions/amp-sidebar/1.0/component.js

```diff
@@ -181,11 +183,58 @@ const Sidebar = forwardRef(SidebarWithRef);
 Sidebar.displayName = 'Sidebar'; // Make findable for tests.
 export {Sidebar};
 
+const ToolbarContext = Preact.createContext(
+  /** @type {SidebarDef.ToolbarContext} */ ({})
+);
+
 /**
  * @param {!SidebarDef.SidebarToolbarProps} props
  * @return {PreactDef.Renderable}
  */
-export function SidebarToolbar({
+export function SidebarToolbar(props) {
+  const context = useMemo(
+    () => ({
+      preactMode: true,
+      domElement: null,
+    }),
+    []
+  );
+
```

Comment:
Decided to pass these two items via context (not props) as I wanted to keep the props namespace clean.  This way any additional props get passed down via `...rest`